### PR TITLE
Item search POST clarification

### PR DESCRIPTION
**Pull request by cholmes**
**Related Issue(s):** #

**Proposed Changes:**

1. Added language to make it clearer how to indicate POST can be used.

**PR Checklist:**

- [ ] This PR is made against the dev branch (all proposed changes except releases should be against dev, not master).
- [x] This PR has **no** breaking changes.
- [ ] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-api-spec/blob/dev/CHANGELOG.md) **or** a CHANGELOG entry is not required.
- [ ] OpenAPI files only: I have run `npm run generate-all` to update the [generated OpenAPI files](https://github.com/radiantearth/stac-api-spec/blob/dev/README.md#openapi-definitions).

### CODE REVIEW

**Comment by cholmes** [comment]
@m-mohr - I hadn't realized you put the `method` thing in until after I submitted #77 I like the approach (hadn't really understood it). But I wasn't sure if you were supposed to add two links or one. My guess is two, so did that here, and tried for a little more clarity on required vs not. If I have it wrong let me know and I can try again.

File: item-search/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Implementing `GET /search` is **required**, `POST /search` is optional, but reco
 
 It is **required** to add a Link to the root endpoint (`/`) with the `rel` type set to `search`
 that refers to the search endpoint in the `href` property,
-with a `type` of `application/geo+json` and a `method` of either `GET` or `POST`.
+with a `type` of `application/geo+json` and a `method` of either `GET`.
```
Comment (m-mohr) [inline item-search/README.md:19]: The "either" is a left-over, right?

Comment (cholmes) [reply item-search/README.md:19]: yes, whoops.

**Review by m-mohr — APPROVED** [review]
With the one requested change good to be merged.

**Pull request merged**